cvat-sdk/gen/templates/openapi-generator/model_utils.mustache

```diff
@@ -1263,27 +1270,21 @@ def validate_and_convert_types(input_value, required_types_mixed, path_to_item,
     input_class_simple = get_simple_class(input_value)
     valid_type = is_valid_type(input_class_simple, valid_classes)
     if not valid_type:
-        if (configuration
```

Comment:
I can still see configuration in the function args, why this check was removed?

Reply:
I want this logic to work the same way as before. Before, `configuration` was always true, so I substituted it with `True`. `True or ...` is equivalent to `True`, so I just removed the condition and the `else` branch.

I reviewed the code of `attempt_convert_item`, and the only place where I found `configuration` to actually be used was inside `deserialize_file`. So I updated the code of `deserialize_file` such that when `configuration` is None, it behaves the same way as if it was `Configuration()`.